scripts/kconfig/kconfigfunctions.py

```diff
@@ -596,7 +597,6 @@ def shields_list_contains(kconf, _, shield):
         "dt_node_int_prop_hex": (dt_node_int_prop, 2, 3),
         "dt_node_str_prop_equals": (dt_node_str_prop_equals, 3, 3),
         "dt_node_has_compat": (dt_node_has_compat, 2, 2),
-        "dt_nodelabel_has_compat": (dt_nodelabel_has_compat, 2, 2),
```

Comment:
I don't think we make any API stability guarantees about these functions, but did you consider printing a deprecation warning if the function is called under the old name, instead of removing it entirely? My concern here is about out of tree users.